noxfile.py

```diff
@@ -19,12 +21,12 @@ def translation(session):
     )
 
 @nox.session(py="3")
-def build(session, autobuild=False):
+def build(session, language='', autobuild=False):
```

Comment:
Can we have 
```suggestion
def build(session, language='en', autobuild=False):
```
and drop the conditional block?

Reply:
Currently, my proposal is to put built docs in different paths according to their language. English already goes to `build/html`, and other languages would go to `build/<lang>/html`. If there is no condition to treat differently English from other languages, English docs will go to `build/en/html`.

Do you see a problem in that?